.github/workflows/github_release.yml

```diff
@@ -0,0 +1,25 @@
+name: Create GitHub Release
+
+on:
+  push:
+    tags:
+      - "v*"  # Trigger when a version tag is pushed (e.g., v1.0.0)
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Create GitHub Release
+        uses: ncipollo/release-action@v1
+        with:
+          tag: ${{ github.ref_name }}
+          name: RuboCop AST ${{ github.ref_name }}
+          bodyFile: relnotes/${{ github.ref_name }}.md
```

Comment:
This won't work, there's no directory like this here. You would need to import the code that generates this from rubocop as well

Reply:
The simplest workaround is to dump the entire changelog there, but it won't be as nice.

Reply:
Yeah, I created a `relnotes` file and included past release files to make this workflow functional. Going forward, this workflow can work based on `relnotes`.

Reply:
Great! It's be nice if we use the same approach across all official RuboCop projects.